randovania/games/game.py

```diff
@@ -96,6 +96,21 @@ class GameGenerator:
     """Use AllJokesDistributor if not using hints."""
 
 
+@dataclass(frozen=True)
+class GameWebInfo:
+    what_can_randomize: Iterable[str] = (
+        "Everything",
```

Comment:
These values fit best as values for Blank Game, instead of being default values.